### PR TITLE
use getWindowSize from GPipe-GLFW

### PR DESCRIPTION
**Pull request by lambdael**
I couldn't compile this project for missing getWindowSize. (Mybe from GLFW-b?)
I also wrapped getWindowSize into GPipe-GLFW and sent a pull request for it.

### CODE REVIEW

File: GPipe-Test/src/Main.hs [hunk 1]
```diff
@@ -71,6 +73,10 @@ main =
     -- Run the loop
     loop win shader makePrimitives uniform 0
 
+-- getWindoSize win = do
```
Comment (SwiftsNamesake) [inline GPipe-Test/src/Main.hs]: typo: getWindowSiz**e**

Comment (lambdael) [inline GPipe-Test/src/Main.hs]: Sorry, these lines were unneeded.
I deleted them and push to my repository.
I'm not sure what should I do next.
Should I create new pull request? Or will this pull request be automatically updated?
This is my first pull request.

**Comment by BlackCapCoder** [comment]
I couldn't compile it even with this patch, so I ended up replacing `getWindowSize` with `return $ Just (3200,1800)`

**Comment by lambdael** [comment]
I found this says we should use `glfwGetFramebufferSize`.
http://www.glfw.org/docs/latest/window_guide.html#window_size
also, GPipe exports framework independent `getFrameBufferSize` function.

So, we should use `getFrameBufferSize` instead of `getWindowSize` here.

**Pull request closed without merging**